packages/terra-clinical-data-grid/src/subcomponents/Row.module.scss

```diff
@@ -1,11 +1,10 @@
 :local {
   .row {
     background-color: var(--terra-clinical-data-grid-row-background-color, #fff);
-    display: flex;
     min-width: 100%;
   }
 
-  .row:nth-child(odd) {
+  .row.striped {
     background-color: var(--terra-clinical-data-grid-row-striped-background-color, #f0f0f0);
   }
 
```

Comment:
The variable `--terra-clinical-data-grid-row-hover-background-color` is currently set to have two different default values:

>   .row.hover {
    background-color: var(--terra-clinical-data-grid-row-hover-background-color, #e2f2fc);
  }

> .row.selected.hover {
    background-color: var(--terra-clinical-data-grid-row-hover-background-color, #b3ddf7);
  }

Perhaps, rename the latter to `--terra-clinical-data-grid-row-selected-hover-background-color` ?

Reply:
+2 nice find, I will update that.